exe/node/Main.hs

```diff
@@ -207,3 +205,8 @@ main = do
     miner nod gos = runGossipT gos . runNodeT nod $ Node.miner
     storage nod =
         hoistStorage (runNodeT nod) (Node.storage Nakamoto.validateBasic)
+
+    runEkg store host port = do
+        tid <- forkEkgServer store host port
+        withException (threadDelay maxBound) $ \(e :: SomeAsyncException) ->
```

Comment:
Let's just hope that one of those cyberpunk hipsters won't find a way to keep a machine going for more than 292471.208677536 years 😂

Reply:
I briefly considered blocking on an empty [M|T]Var, but that steampunk compiler is too smart ;)